### ISSUE TITLE
perference of training NRTR

### ISSUE THREAD

**Issue by lantx5** [open-mmlab/mmocr#494]
![image](https://user-images.githubusercontent.com/73979084/133575241-462b907c-b626-4f4e-be91-de0a8d00e519.png)

I just changed lr of configs.
My word_accuracy of NRTR is very low. Should I add a vocabulary somewhere? or I need to change something

**Comment by gaotongxiao** [comment]
I couldn't make any specific suggestions on that. Apparently, the model was just making too many mistakes on the letter case in some test sets, which shouldn't be a big problem as the reported metric `word_acc_ignore_case_symbol` still makes sense to me. It also has nothing to do with the dictionary which already contains all upper and lower case letters. Fine-tuning it for more iterations may help?

**Comment by lantx5** [comment]
Could you tell me in which document these evaluation criteria are defined?    thanks

**Comment by gaotongxiao** [comment]
https://github.com/open-mmlab/mmocr/blob/main/mmocr/core/evaluation/ocr_metric.py
